packages/pricing/src/mappings/coingecko.mapping.ts

```diff
@@ -0,0 +1,60 @@
+    "0xF9c53268e9de692AE1b2ea5216E24e1c3ad7CB1E": "idexo-token",
+    "0x6B175474E89094C44Da98b954EedeAC495271d0F": "dai",
+    "0x5F64Ab1544D28732F0A24F4713c2C8ec0dA089f0": "dextf",
+    "0xeEAA40B28A2d1b0B08f6f97bB1DD4B75316c6107": "govi",
+    "0x5f98805A4E8be255a32880FDeC7F6728C6568bA0": "liquity-usd",
+    "0x6982508145454Ce325dDbE47a25d4ec3d2311933": "pepe",
+    "0xd38BB40815d2B0c2d2c866e0c72c5728ffC76dd9": "symbiosis-finance",
+    "0xDDdddd4301A082e62E84e43F474f044423921918": "rhinofi",
+    "0xBe9895146f7AF43049ca1c1AE358B0541Ea49704": "coinbase-wrapped-staked-eth",
+    "0xBBBbbBBB46A1dA0F0C3F64522c275BAA4C332636": "zkspace",
+    "0xD31a59c85aE9D8edEFeC411D448f90841571b89c": "sol-wormhole",
+    "0xB64ef51C888972c908CFacf59B47C1AfBC0Ab8aC": "storj",
+    "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2": "weth",
+    "0x582d872A1B094FC48F5DE31D3B73F2D9bE47def1": "the-open-network",
+    "0xfAC77A24E52B463bA9857d6b758ba41aE20e31FF": "lsdx-finance",
+    "0xf939E0A03FB07F59A73314E73794Be0E57ac1b4E": "crvusd",
+    "0x7f39C581F595B53c5cb19bD0b3f8dA6c935E2Ca0": "wrapped-steth",
+    "0x95aD61b0a150d79219dCF64E1E6Cc01f0B64C4cE": "shiba-inu",
+    "0xA91ac63D040dEB1b7A5E4d4134aD23eb0ba07e14": "bella-protocol",
+    "0x111111111117dC0aa78b770fA6A738034120C302": "1inch",
+    "0xA487bF43cF3b10dffc97A9A744cbB7036965d3b9": "deri-protocol",
+    "0x163f8C2467924be0ae7B5347228CABF260318753": "worldcoin-wld",
+    "0x7448c7456a97769F6cD04F1E83A4a23cCdC46aBD": "maverick-protocol",
+    "0xa41d2f8Ee4F47D3B860A149765A7dF8c3287b7F0": "syncus",
+    "0xC91a71A1fFA3d8B22ba615BA1B9c01b2BBBf55ad": "zigzag-2",
+    "0x18084fbA666a33d37592fA2633fD49a74DD93a88": "tbtc",
+    "0xA1290d69c65A6Fe4DF752f95823fae25cB99e5A7": "kelp-dao-restaked-eth",
+    "0x0a77eF9bf662D62Fbf9BA4cf861EaA83F9CC4FEC": "x-world-games",
+    "0x72e364F2ABdC788b7E918bc238B21f109Cd634D7": "metaverse-index",
+    "0x514910771AF9Ca656af840dff83E8264EcF986CA": "chainlink",
+    "0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9": "aave",
+    "0x5C1d9aA868a30795F92fAe903eDc9eFF269044bf": "changer",
+    "0x88dF592F8eb5D7Bd38bFeF7dEb0fBc02cf3778a0": "tellor",
+    "0xbf5495Efe5DB9ce00f80364C8B423567e58d2110": "renzo-restaked-eth",
+    "0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0": "matic-network",
+    "0x4691937a7508860F876c9c0a2a617E7d9E945D4B": "woo-network",
+    "0x62D0A8458eD7719FDAF978fe5929C6D342B0bFcE": "beam-2",
+    "0xC9fE6E1C76210bE83DC1B5b20ec7FD010B0b1D15": "fringe-finance",
+    "0xfFffFffF2ba8F66D4e51811C5190992176930278": "furucombo",
+    "0xd5F7838F5C461fefF7FE49ea5ebaF7728bB0ADfa": "mantle-staked-ether",
+    "0xe2353069f71a27bBbe66eEabfF05dE109c7d5E19": "bonsai3",
+    "0xD33526068D116cE69F19A9ee46F0bd304F21A51f": "rocket-pool",
+};
+
+export const idToAddress: Record<CoingeckoId, Address> = Object.fromEntries(
+    Object.entries(addressToId).map(([address, id]) => [id, address]) as [CoingeckoId, Address][],
+);
```

Comment:
Might want to validate that the sizes of the resulting object is not changed? To avoid something like this:
```
{ "a": "foo", "b": "foo" }
```
Being transformed to this:
```
{ "foo": "b" }
```

Reply:
would be a nice check but I don't think is needed here. since it's done only once, not as part of function call or smth